instrumentation/othttp/common.go

```diff
@@ -28,6 +28,14 @@ const (
 	WriteErrorKey = kv.Key("http.write_error") // if an error occurred while writing a reply, the string of the error (io.EOF is not recorded)
 )
 
+// Server HTTP metrics
+const (
```

Comment:
It looks like https://github.com/open-telemetry/opentelemetry-specification/pull/657 is approaching agreement. To follow that proposal, the request count is implied by the ServerLatency's Count. 

(As a theoretical question, I think it's a useful exercise to think through the reasons we do not suggest adding request and response content length as labels on the duration event? The answer is that these translate into high cardinality. Whether the downstream system can or cannot handle the cardinality isn't the question, it's the fact that the SDK itself does a lot of extra work because it aggregates by the full label set before processing and exporting data.)

Reply:
I wasn't clear. I think we should remove the RequestCount metric. It is implied by ServerLatency.Count.

Reply:
kk, will remove